model/src/cfd.rs

```diff
@@ -2921,6 +2948,64 @@ mod tests {
         assert_eq!(is_liquidation_price, should_liquidation_price);
     }
 
+    #[test]
+    fn correctly_calculate_hours_to_extend_in_rollover() {
+        let settlement_interval = SETTLEMENT_INTERVAL.whole_hours();
+        for hour in 0..settlement_interval {
+            let event_id_in_x_hours = BitMexPriceEventId::with_20_digits(
+                OffsetDateTime::now_utc() + Duration::new(hour * 60 * 60, 0),
```

Comment:
```suggestion
                OffsetDateTime::now_utc() + hour.hours(),
```

Might be more readable.